tests/python/pants_test/test_base.py

```diff
@@ -362,6 +362,10 @@ def isolated_local_store(cls):
   def build_root(self):
     return self._build_root()
 
+  @property
+  def build_file_parser(self):
```

Comment:
One note: the new engine does not expose a "BuildFileParser" interface, so I believe that usage of this class actually represents dead (or at least, alive-only-in-tests) code.

Reply:
Yeah, I noticed that, but didn't want to perturb too far...

Reply:
Should I just kill this?

Reply:
If you have the appetite to, it would definitely be appreciated! Not sure how far reaching it is.